packages/server/src/features/tips/routes.js

```diff
@@ -32,4 +32,16 @@ router.post(
   tipsController.postTipComment
 );
 
+router.get(
+  "/tips/:blockHeight(\\d+)_:tipHash/rates",
+  maybeAuth,
+  tipsController.getRates,
+);
+
+router.get(
+  "/tips/:blockHeight(\\d+)_:tipHash/ratestats",
+  tipsController.getRateStats,
+);
```

Comment:
I think we can remove tip rates. Not so necessary for small tip expenses currently.